api/proto/info/info.proto

```diff
@@ -241,6 +241,16 @@ message ZInfoAppInstance {
   string uuid = 1;
   string name = 2;
   string domainName = 3;
+  string taskid = 4;
+}
+
+// ZInfoDeviceTasks - send a summary of tasks so that controller knows
+// how many app instance tasks are still present on the device. Controller
+// can then use it to wait for an app to be fully deleted etc.
+message ZInfoDeviceTasks {
+  string id = 1;
+  string name = 2;
```

Comment:
What's the difference between ID and name?

Reply:
What do you want me to add in the proto file? I thought it is self-explanatory that id is representing the task's unique id and the name is representing the task name in the containerd world. I am not sure what can I add to the proto file. Can you please help me in understanding the requirement here?

Reply:
That's literally my question -- what is that unique ID? I don't understand it since it doesn't seem to be a containerd concept, is it @zed-rishabh ?